spot_wrapper/spot_arm.py

```diff
@@ -496,6 +496,10 @@ def hand_pose(
 
             arm_cartesian_command = arm_command_pb2.ArmCartesianCommand.Request(
                 root_frame_name=ref_frame,
+                wrist_tform_tool=geometry_pb2.SE3Pose(
```

Comment:
@llee-bdai Is this the function that you recently fixed? I am trying to figure out how much of a breaking change this is.